tests/test_typing.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from pydantic.typing import Literal
+from pydantic.validators import all_literal_values
+
+
+@pytest.mark.skipif(not Literal, reason='typing_extensions not installed')
+def test_all_literal_values():
```

Comment:
This doesn't need a new file, move it to `test_utils.py` where other tests for `typing.py` live.

(In future `test_utils.py` might need be be split, but that should be done properly not in an ad hoc way.)

Reply:
Done